DataFormats/CSCDigi/src/CSCCLCTDigi.cc

```diff
@@ -81,7 +84,7 @@ void CSCCLCTDigi::clear() {
   }
 }
 
-int CSCCLCTDigi::getKeyStrip(int n) const {
+uint16_t CSCCLCTDigi::getKeyStrip(const uint16_t n) const {
   // 10-bit case for strip data word
   if (compCode_ != -1 and n == 8) {
```

Comment:
This is for Run 3 only, I guess?  Just to confirm: the "compCode" has never been used in Run 2, i.e. it was unused in firmware, and was always set to -1 in software?  (See getCompCode() function above.)

Reply:
Yes, correct. I suppose we can update this function so that for Run-1/2 it always returns the half-strip, whatever the argument.

Reply:
I think that's already what it does, no?  See below, "// 8-bit case for strip data word (all other cases)"

Reply:
Yes, although we could strengthen it with an `isRun3()` check.